.circleci/config.yml

```diff
@@ -62,6 +62,10 @@ commands:
           name: Install make
           command: |
             sudo apt-get install make
+      - run:
+          name: Install clj-kondo
+          command: |
+           sudo curl -sLO https://raw.githubusercontent.com/clj-kondo/clj-kondo/master/script/install-clj-kondo && sudo chmod +x install-clj-kondo && sudo ./install-clj-kondo
```

Comment:
This would be a good chance to review whether https://github.com/clojure-emacs/cider-nrepl/pull/693 is a desirable technique

Reply:
I think it is, so you're welcome to apply it here as well.